### ISSUE TITLE
`Active` defining/rethinking path to active

### ISSUE THREAD

**Issue by Ryun1** [cardano-foundation/CIPs#927]

**Comment by Ryun1** [comment]
> How do we define active in an ecosystem where there are multiple implementations?

Im not sure we can,
unless a implementation explicitly signals their interest being included in the `active` status,

**Comment by perturbing** [comment]
I've been reflecting on this issue and believe we do not need to rephrase or change CIP-0001. The main challenge seems to lie in how we, as CIP editors, approach marking proposals as active.

To address this, I propose adopting a more critical and evidence-based approach. For example:

- How stable is the CIP in question?
- Is there tangible evidence of stability, such as implementation success or community adoption?

A relevant case is CIP-1694, which was marked active despite mainnet not yet passing the Plomin hardfork. According to CIP-0001:

> For changes to the Cardano protocol, a CIP becomes 'Active' by being live on the Cardano mainnet;

This suggests it should not yet be active. However, one could argue that it is production-ready, as the node for the hardfork has been released—highlighting that this is not an exact science.

To improve consistency, I suggest that any status change be accompanied by clear, criteria-based arguments measured against the wording of CIP-0001. This will enhance transparency, align our decisions with intended standards, and encourage thoughtful deliberation.

Note that after the Plomin hardfork, we will be able to mark some CIP's active (the new bitwise and ripemd builtins for example).

**Comment by rphair** [comment]
@perturbing I agree 100% with https://github.com/cardano-foundation/CIPs/issues/927#issuecomment-2575141576.  Since CIP-0001 is not inaccurate as it is now, I suggest we create another Wiki page in this section (a new outline point `302`: https://github.com/cardano-foundation/CIPs/wiki/3.-CIPs-for-Editors

... with the editorial decision-making process behind assigning `Active` status, expressed as concisely as possible from what's been written above + anything further agreed upon here before closing this issue.

**Comment by rphair** [comment]
p.s. after discussion in the last hour's CIP meeting, I will create that Wiki page with a digest of the postings in this issue so far in a new Wiki page located above.  This will be a living document that all editors can use & update whenever the application of `Active` status is subject to any of the above uncertainties.

**Comment by rphair** [comment]
another point for discussion: how can we {flag, log, schedule, automate} progressions from `Proposed` to `Active` when new information like this is posted in the repository or community?
- https://github.com/cardano-foundation/CIPs/pull/641#issuecomment-2932049424

**Comment by rphair** [comment]
I've noted this issue to include in the discussion at Discord channel [#cip-processes](https://discord.com/channels/971785110770831360/1375190972476035143) that has formed recently & I'll feed back some notes here if & when that discussion clarifies whatever changes we need to make.

I do think we need to hold off on any CIP-0001 and Wiki updates until we can bring in some developers & implementors into this discussion and I think that "working group" will be a good chance to do this.

**Comment by rphair** [comment]
OK @Ryun1 @perturbing I think we can finally get ahead of this issue by recommending a supportive vocabulary in this discussion (cc @Crypto2099):
* https://github.com/cardano-foundation/cf-cip-frontend/discussions/55

... because, as we've covered in meetings & discussion already, the interpretation of `Active` status may well depend on what kind of software / community a CIP is affecting.  Therefore we should have some new "tags" that give us enough information to make that interpretation correctly. 🤔

**Comment by Ryun1** [comment]
@rphair 

> I think we can finally get ahead of this issue by recommending a supportive vocabulary in this

yes! I agree

happy to keep this issue open for the moment though
as I feel its certainly an longstanding ongoing discussion/issue

**Comment by rphair** [comment]
yes @Ryun1 it needs to be kept open... FYI @aldo555 has closed all our pending issues, including those that apply to this purpose, because "the demo site has gone into production" (https://github.com/cardano-foundation/cf-cip-frontend/issues/53#issuecomment-3323542521) and indeed we can see it all now on `cips.cardano.org` & thanks @aldo555 it's magnificent to see it live. 🙏

There were still some pending unresolved items — including search functionality asked for in issues closed in the round even before this one — so we've lost our chance to discreetly add some finer points into the released version & it may take some time to go through another round of feature request & release.

Mainly: in those prior issues I think we were moving towards the a vital improvement of a Query URL such as the kind that's supported on most sites with embedded search (from Amazon to the [Cardano Explorers URI syntax](https://github.com/cardano-foundation/developer-portal/pull/1538#issuecomment-3030844444)).  Once a composite query of appropriate categories & tags is supported — which would also address this requirement:
* https://github.com/cardano-foundation/CIPs/issues/863#issuecomment-3323363845

— a properly provisioned web site should be able to pop up a list of matching CIPs and CIPs for these "derived" classifications.

Unfortunately neither search page URLs nor query string entry into the search parameters is supported: search terms can only be entered by hand into the tiny little world of that dialogue box... not passable in from the outside in any way that I can tell.  TL;DR pages of the new site display _only_ CIPs and CPSs and nothing else, and _not_ search results.

I'll link back here as soon as I can get a summary of any pending requirements (re)posted as issues on the other repo: right now I think we are just missing URL query with search result pages.  If @aldo555 will accept a new issue for these, then the rest is up to us @Ryun1 @Crypto2099 @perturbing + the CIP community to progress & conclude the [extended vocabulary](https://github.com/cardano-foundation/cf-cip-frontend/discussions/55) discussion.

**Comment by rphair** [comment]
note it seems I "pinned this issue" (looking at the traces above) but that was not intentional: must have clicked some GitHub link by accident.

**Comment by rphair** [comment]
@Ryun1 @perturbing as we approach 2 years since this issue was posted, I believe we have proven by default that the only way to define aspects of progress toward `Active` status — likely the more difficult way in which _Path to Active_ depends on category — is by supporting one or more tags applicable to that category.
* Therefore I propose that we leave this issue closed and instead try to **shape any such definitions into a "tag" list** the community can observe & work with in reporting on CIPs and CPSs: so I've absorbed this issue into a top-level issue as described here: https://github.com/cardano-foundation/CIPs/issues/1076#issuecomment-5389019088
* Since tags — once sparingly created by editors — can be easily added by authors, routinely checked by validators, and readily displayed on `cips.cardano.org` (https://github.com/cardano-foundation/cf-cip-frontend/discussions/55#discussioncomment-14427367) we will have a firm solution for this if & when all the aspects of _Path to Active_ subtlety identified above can be defined as tags.

The other aspect of the discussion above is the much simpler case of establishing a "checklist" that we would need to certify `Active` status for each category: e.g. for a `Token` category CIP there should be implementation in`X` wallet projects, `Y` dApps, `Z` analytics services, etc.
* Though a maybe simpler aspect of this issue — since it would only require these draft requirements for each `Category:` to be added to CIP-0001 or that category's own CIP (i.e. `Ledger` & `Plutus`) — I don't believe this will be possible to standardise into CIP-0001 because expectations for volatile categories like `Wallet`, `Metadata`, and `Tools` have varied by time, ecosystem participation, and our CIP Editors' roster.

Therefore I propose that we close off this second aspect of the problem as it would produce a section of CIP-0001 which would be unmaintainable & eventually diverge from our own practice as it continues to adjust to ecosystem expectations.
* Alternatively, if another editor has a proposal of specific _Path to Active_ language for particular categories, I would welcome a PR about it & think that we would then get the best review on such requirements by presenting that PR to our usual reviewers... while further comments in this issue will not get any such review & response.
